transport-classes-epoll/src/main/java/io/netty5/channel/epoll/EpollDatagramChannel.java

```diff
@@ -700,27 +677,22 @@ private static void releaseAndRecycle(Object buffer, RecyclableArrayList packetL
         }
     }
 
-    private static boolean processPacket(ReadSink readSink,
+    private static void processPacket(ReadSink readSink,
                                          int attemptedBytesRead, int bytesRead, AddressedEnvelope<?, ?> packet) {
         // Avoid signalling end-of-data for zero-sized datagrams.
-        return readSink.processRead(attemptedBytesRead, Math.max(1, bytesRead), packet);
+        readSink.processRead(attemptedBytesRead, Math.max(1, bytesRead), packet);
```

Comment:
These comments… `// Avoid signalling end-of-data for zero-sized datagrams.` – I'm not sure we still need that `Math.max(1, bytesRead)`, but we can address that in a follow-up.

Reply:
yep lets fix it while we are here.